README.md

```diff
@@ -1389,6 +1389,12 @@ Language: English
 Notable Channels: `#wordedit`, `#worldguard`, `#craftbook`, `#commandbook`, `#enginehub`, `#api`, `#dev-team`, `#git-log`, `#showcase` \
 Language: English
 
+<img align="left" height="94px" width="94px" alt="Server Icon" src="images/server_icons/intellectualsites.png" />
+
+[__IntellectualSites__](https://discord.com/invite/KxkjDVg) [<img height="16px" width="16px" alt="Homepage URL" src="images/badges/homepage.png">](https://intellectualsites.com/) [<img height="16px" width="16px" alt="Git Repository" src="images/badges/git.png">](https://github.com/IntellectualSites) \
+Notable Channels: `#announcements`, `#info`, `#fawe-support`, `#hv-general-chat`, `#is-web`, `#programming` \
```

Comment:
@N0tMyFaultOG Thanks for adding your community. You can remove the `#announcements, #info`, since they are found in every server, maybe replace them with `#fawe-developer-chat`? Everything else looks good.

Reply:
Right, seems so. Changed 👍